qa/pull-tester/rpc-tests.sh

```diff
@@ -143,6 +143,7 @@ testScripts=(
   'sc_getcertmaturityinfo.py'
   'sc_big_commitment_tree.py'
   'sc_big_commitment_tree_getblockmerkleroot.py'
+  'shieldedpooldeprecation_rpc.py'
```

Comment:
With the new CI workload rebalancer we need to specify the execution time  for each new `.py` we add to this list.
To have a rough estimate of the execution time on the CI, I suggest to measure it locally, then multiply by 1.3 for the Linux and 2.0 for MacOS. Alternatively, any placeholder value would suit well, at least for the first few runs.
Suggested change:
`'shieldedpooldeprecation_rpc.py',60,90`

Reply:
Agree, modified with slight increase of test time `'shieldedpooldeprecation_rpc.py',450,600` (due to additional test steps).